modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+[id="authenticating-with-the-guest-user_{context}"]
+= Authenticating with the Guest user
+
+To explore {product-short} features, you can skip configuring authentication and authorization. 
+After the default installation, you can log in as a Guest user and access all the features.
+
+.Prerequisites
+* You installed {product-short} by using default values.
```

Comment:
With 1.3 the default installation does not work OOTB. @PatAKnight and @gustavolira to confirm:

```suggestion
* You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html/administration_guide_for_red_hat_developer_hub/assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration], and have sufficient permissions to modify it.
```